assembly/collections/__tests__/persistentMap.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { PersistentMap } from "../persistentMap";
+
+describe("Persistent Map tests", () => {
+  it("empty map", () => {
+    const map = new PersistentMap<string, string>("my_map");
+    assert<bool>(map.size() === 0, "empty map must have 0 size");
```

Comment:
we are now using as-pect test framework, it is more convenient to right test, see the doc here: https://as-pect.gitbook.io/as-pect/

it's ok for this PR if we stay like this with asserts